docs/dcp2_operating_procedures.rst

```diff
@@ -0,0 +1,253 @@
+
+2)  Prepare a PR against the `dcp2`_ repository, updating the `DCP/2 System
+    Design`_.
+
+3)  Initiate a `Pull request review`_ of your PR. Once the PR has been approved
+    by the DCP, move on to the next step. Don't merge the PR just yet.
+
+4)  Prepare a PR against the ``develop`` branch of the `metadata-schema`_
+    repository. It is OK to perform this step concurrently with the preceding
+    steps.
+
+5)  Initiate a `Pull request review`_ of your PR. Once the PR has been approved
+    by the DCP, move on to the next step. Don't merge the PR just yet.
+
+    Note that the above steps can be performed for several independent schema
+    changes in parallel.
+   
+6)  Merge the `metadata-schema`_ repository PR into the ``develop`` branch and
+    from there into the ``staging`` branch.
+
+7)  Prepare a PR against the `Test metadata`_ in the `schema-test-data`_
+    repository. Don't merge the PR just yet.
+
+8)  Initiate a `Pull request review`_ of your PR. Once the PR has been approved
+    by the DCP, move on to the next step. As part of the review, the Azul team
+    updates their unit tests to validate the updated test metadata. If a test
+    fails, they either improve the test or the team member reviewing the PR
+    requests the necessarty changes to the test metadata.
+
+9)  Prepare a PR against the schema repository, targetting the main branch.
+
+10) Initiate a `Pull request review`_ of your PR. The review should be quick
+    because reviewers will already have seen the changes while reviewing the PR
+    against the ``develop`` branch. Reviewers should not request any substantial
+    changes at this stage unless last minute considerations absolutely require
+    it. Once the PR has been approved by the DCP, move on to the next step.
+
+11) Merge all PRs. You may squash and/or rebase a PR before merging it but that
+    should not affect the diff between the base and head commits of the PR
+    branch, except for housekeeping like resolving merge conflicts or adjusting
+    the host name of schema URLs in PRs against the `schema-test-data`_
+    repository. No semantic changes may be introduced to a PR after it has been
+    approved by the DCP. All such housekeeping should be done on the PR branch
+    prior to merging it.
+
+
+Test metadata
+-------------
+
+Every change to the schemas in the `metadata-schema`_ repository must be
+accompanied by a matching change to the test metadata in the `schema-test-data`_
+repository. Both changes must be done as PRs following the `Pull request
+review`_  process. 
+
+The test metadata should excercise all graph shapes in actual use. 
+
+The schema references (``describedBy``) in all metadata documents in the main
+branch of `schema-test-data`_ repository must match the schemas on the
+`master` branch of the `metadata-schema`_ repository. Pull requests against the 
+
+
+Pull request review
+-------------------
+
+As a PR author
```

Comment:
I do mean PR author. Note the

> The schema change author can delegate authorship of PRs to other members of the DCP consortium.

in that section.

Reply:
I think this is just framing Marion's question differently -- 1. whats a scenario of when someone is the schema change author and not the PR author -- and what is the value of separating those roles? 2. if one is a schema change author and delegated authorship of PRs to other members of the DCP -- then what is the role/responsibility of the schema change author in that scenario?

Reply:
I was trying to accommodate what I think I observed during the organic CGM effort. Different people were authoring different PRs. I'd be curious to what @ESapenaVentura and @amnonkhen think about this.

Reply:
If I understood it correctly, we're talking here about responsibilities, right? e.g. A PR may be opened by person A but person B may be the schema change author and therefore is responsible for requests.

I do think that this distinction is useful, especially in cases where e.g. a person starts a PR for a schema change and then due to some unexpected circumstances is not reliable to lead the effort of responding to requests/changes for the next 2 weeks.

Most of the times person A will be both schema change author and PR author, but preparing for different case scenarios seems fine to me.

It may need some clarification if the distinction between both roles is not clear to the readers, though.

Reply:
As written, the spec doesn't accommodate the possibility of PR authors changing mid-PR. It only accommodates the possibility that the PRs involved in a schema change are authored by different people. It requires that one person is responsible for the schema change as whole. This is also often referred to as a *sponsor*. We need one person to pull the strings and be accountable for every schema change.

Reply:
Fixed